### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by VulcanWM** [VulcanWM/munity#14] [Announcements]
# Munity

This is the release of the game Munity!
All the instructions are in the main page of the website and on README.md.

<hr /><em>This discussion was created from the release <a href='https://github.com/VulcanWM/munity/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by transcental** [comment]
Awesome. Really cool website/game.

I'm not very good at it...
